jenkins/common.groovy

```diff
@@ -109,10 +115,13 @@ def build(String ref, dockerContainer, Boolean contentOnlyBuild) {
       for (int i=0; i<VAGOV_BUILDTYPES.size(); i++) {
         def envName = VAGOV_BUILDTYPES.get(i)
         def buildDetails = buildDetails(envName, ref)
+        def drupalAddess = DRUPAL_ADDRESSES.get(envName)
```

Comment:
Also, I think for our vets-website builds, we want our dev environment to point to their staging environment. That's what we discussed with them before.

Reply:
Should I just get rid of the drupal address stuff and leave it to devs to put it in the code?

Reply:
I think it's useful for Jenkins to set the address, since it has to use the aws urls and local devs need to use *.cms.va.gov. But it's also sort of confusing to have the env mapping in both the Jenkins code and the regular JS code. @ncksllvn, do you have thoughts?